travis/upload_build_results_to_gcs.py

```diff
@@ -216,6 +249,14 @@ def main(argv):
     gcs_bucket = gcs_client.get_bucket(os.environ[_GCS_BUCKET])
     gcs_build_results_dir = _GetGCSBuildResultsDir()
     _UploadBuildResults(gcs_bucket, gcs_build_results_dir)
+
+    # Upload the generated OpenAPI description and the generated documentation.
+    if len(flags.FLAGS.openapi_json_dir) > 0:
+      gcs_bucket_openapi = (
+        gcs_client.get_bucket(os.environ[_GCS_BUCKET_OPENAPI])
+      )
+      _UploadOpenApiJson(gcs_bucket_openapi, "openapi_description")
+      _UploadDocumentation(gcs_bucket_openapi, "documentation")
```

Comment:
Should these two values be hardcoded? Everywhere else they seem to be passed either by flags or by environmental variables.

Reply:
These are the names of the directories used in the GCS bucket to upload the VM-local description JSON and documentation HTML, the previous flags and environment variables were used for the paths of the VM-local files. We can definitely use flags for the GCS bucket folders as well, but when I wrote the script I thought they should be relatively static. Please let me know if you want me to get their names through flags as well.

Reply:
Makes sense, they can be static.